tests/test_taxon_concat_tnrs.py

```diff
@@ -60,7 +59,9 @@ def test_check_taxon_real(self):
         self.CsvCreatePicturae.taxon_check_real()
         # assert statements
         self.assertEqual(len(self.CsvCreatePicturae.record_full.columns), 15)
-        # 2 as the genus level hybrid Serapicamtis and the mispelled Castilloja should fail
+
+        # 2 rows left as the genus level hybrid Serapicamptis and the mispelled "Castilloja" should fail
```

Comment:
test cases for tnrs, testing whether ouput data is right length, size, and has correcto columns. 2 cases should fail on purpose